clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -761,7 +790,7 @@ public void assign(List<TopicPartition> partitions) {
     }
 
     /**
-     * Fetches data for the topics or partitions specified using one of the subscribe/assign APIs. It is an error to not have
+     * Fetch data for the topics or partitions specified using one of the subscribe/assign APIs. It is an error to not have
```

Comment:
Also, this is probably as good a time as any to document the fact that poll() will not respect the timeout during a rebalance.